melosys-eessi-app/src/main/java/no/nav/melosys/eessi/config/KafkaConsumerConfig.java

```diff
@@ -60,13 +69,30 @@ public KafkaConsumerConfig(@Value("${melosys.kafka.consumer.groupid}") String gr
     @Bean
     public KafkaListenerContainerFactory<ConcurrentMessageListenerContainer<String, SedHendelse>> sedMottattListenerContainerFactory(
             KafkaProperties properties) {
-        return sedListenerContainerFactory(properties, recordFilterStrategySedMottatt());
+        return sedListenerContainerFactory(properties, recordFilterStrategySedMottatt(), new SeekToCurrentErrorHandler());
     }
 
     @Bean
     public KafkaListenerContainerFactory<ConcurrentMessageListenerContainer<String, SedHendelse>> sedSendtListenerContainerFactory(
-            KafkaProperties properties) {
-        return sedListenerContainerFactory(properties, recordFilterStrategySedSendt());
+            KafkaProperties properties
+    ) {
+        return sedListenerContainerFactory(
+            properties,
+            recordFilterStrategySedSendt(),
+            new SeekToCurrentErrorHandler(sedSendtRecoverer(), new FixedBackOff(5000L, 9L))
```

Comment:
SeekToCurrentErrorHandler:
>An error handler that seeks to the current offset for each topic in the remaining records. Used to rewind partitions after a message failure so that it can be replayed.

Default så har den maks 9 retries uten delay mellom.

`new FixedBackOff(5000L, 9L)`: 5 sekunder delay mellom evt prosessering etter feil, og maks 9 retries.
`sedSendtRecoverer` blir kalt om det fortsatt kastes feil etter retry.

Reply:
👍  Så da forsøker vi å resende SED som feilet maks 9 ganger, med 5 sek delay hvor du logger "Kan Ikke behandle SED" hver gang? Og dersom det ikke går etter niende forsøk så sier vi at det er feil?

Reply:
Vi prøver ikke å resende SED her, her mottar vi hendelse om at en SED er sendt, som vi videre bruker for å journalføre selve dokumentet 😄 

Selve loggingen (`sedSendtRecoverer`) vil kun bli kalt om alle retryene har feilet også 😄